### DISCUSSION TITLE
关于mix-space的安装

### DISCUSSION THREAD

**Discussion by Mxiaoju** [mx-space/kami#1676] [Q&A]
很不幸，我没有成功配置mix-space，
使用一键docker部署会提示 bash: syntax error near unexpected token `newline' 
使用脚本部署会提示 build.sh: line 20: uuidgen: command not found uuidgen 未安装 
怎么办呢，我目前接触linux比较少，也没能在bing上搜索到解决办法。
可以提供一点帮助吗

**Comment by ttimochan** [comment]
你可以尝试按照手动安装的部分。
